src/sentry/static/sentry/less/shared-components.less

```diff
@@ -2546,22 +2546,6 @@ ul.radio-inputs {
 * Well
```

Comment:
Does this comment block still apply?

Reply:
Yes. `blankslate` is still used on a few wells that contain font-icons.